corehq/apps/cloudcare/tests/test_api.py

```diff
@@ -10,9 +14,8 @@
 from corehq.apps.users.util import format_username
 from corehq.apps.cloudcare.api import get_filtered_cases, CaseAPIResult, CASE_STATUS_OPEN, CASE_STATUS_ALL,\
     CASE_STATUS_CLOSED
-import uuid
 
-TEST_DOMAIN = "test-domain-uCSArw4SkbM6F04"
```

Comment:
we should never use capital letters in domain names. it throws off validation since we normalize the domain names when we save them